src/main.ts

```diff
@@ -0,0 +1,50 @@
+import { match, MatchFunction, MatchResult } from "path-to-regexp";
+import { handleImportMap } from "./handleImportMap";
+import { notFoundResponse } from "./notFound";
+
+addEventListener("fetch", (evt: FetchEvent) => {
+  evt.respondWith(handleRequest(evt.request));
+});
+
+const routeHandlers: RouteHandlers = {
+  "/:orgKey/:importMapName.importmap": handleImportMap,
+};
+
+const routeMatchers: RouteMatchers = Object.entries(routeHandlers).map(
+  ([path, handler]) => [match(path), handler]
+);
+
+async function handleRequest(request: Request) {
+  const requestUrl = new URL(request.url);
+
+  let routeHandler: RouteHandler | undefined,
+    matchResult: MatchResult | false = false;
+
+  // Find which route handler to call for this request
+  // We don't have express to do this automatically for us, but are
```

Comment:
I'm trying to understand why we don't have express again.  Where does this code live?

Reply:
It's inside of cloudflare workers. See https://developers.cloudflare.com/workers/